src/isIsogram.test.js

```diff
@@ -6,4 +6,36 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for an empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should return true for a word without repeating letters', () => {
+    expect(isIsogram('isogram')).toBe(true);
+    expect(isIsogram('word')).toBe(true);
+    expect(isIsogram('abcdefghijklmnopqrstuvwxyz')).toBe(true);
+  });
+
+  it('should return true for words with repeating letters', () => {
```

Comment:
The description for this test case should say 'should return true for words with mixed case and no repeating letters', as it's testing mixed-case words without repeating letters.